src/Domain/LeanCode.DomainModels.EF/PropertyBuilderExtensions.cs

```diff
@@ -61,4 +62,38 @@ public static PropertyBuilder<SId<T>> IsTypedId<T>(this PropertyBuilder<SId<T>>
             .IsFixedLength()
             .ValueGeneratedNever();
     }
+
+    public static PropertyBuilder<TId> IsTypedId<TBacking, TId>(this PropertyBuilder<TId> builder)
```

Comment:
We could expose methods for configuring conventions as well.

Reply:
DOne